src/components/SearchCard/SearchCardHeader.js

```diff
@@ -0,0 +1,143 @@
+import { Box, Heading } from 'grommet'
+import { Button } from 'components/shared/Button'
+import { IconBadge } from 'components/shared/IconBadge'
+import { InlineMessage } from 'components/shared/InlineMessage'
+import { Link } from 'components/shared/Link'
+import { Pill } from 'components/shared/Pill'
+import { Row } from 'components/shared/Row'
+
+/* TEMPORARY the following prop is added for demo purpose
+prop name: 'status' 
+   - ''(default)
+   - added
+   - processing 
+   - add_remaining
+   - not_supporte
+   - request
+   - unavailable
+*/
+
+// CTA for My Dataset based on a status
+const DatasetCTA = ({ status }) => (
```

Comment:
Since you have already broken out the `SearchCard` into multiple components I think it would make sense to break this one out as well.

Reply:
Renamed it to `SearchCardHeaderCTA` and add its own file, thank you!